src/Core/Mews.Fiscalizations.Core/Model/Countries/EuropeanUnionCountry.cs

```diff
@@ -5,15 +5,18 @@ namespace Mews.Fiscalizations.Core.Model
 {
     public class EuropeanUnionCountry
     {
-        internal EuropeanUnionCountry(string alpha2Code, Regex taxpayerNumberPattern)
+        internal EuropeanUnionCountry(string alpha2Code, Regex regexWithCountryCodePrefix, Regex regexWithoutCountryCodePrefix)
         {
             Alpha2Code = alpha2Code;
-            TaxpayerNumberPattern = taxpayerNumberPattern;
+            RegexWithCountryCodePrefix = regexWithCountryCodePrefix;
+            RegexWithoutCountryCodePrefix = regexWithoutCountryCodePrefix;
```

Comment:
Right, but we need to name it better. This class is a country. It's not obvious, what the regex is for.
So maybe `taxpayerNumberPattern` and `taxpayerNumberPatternWithoutCountryCodePrefix`?